### PR TITLE
A few improvements

### PR DESCRIPTION
**Pull request by pedroigor**

### CODE REVIEW

File: quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakProcessor.java [hunk 1]
```diff
@@ -102,7 +102,7 @@ void configureHibernate(KeycloakRecorder recorder, HibernateOrmConfig config, Li
      * 
      * @param recorder
      */
-    @Record(ExecutionTime.STATIC_INIT)
+    @Record(ExecutionTime.RUNTIME_INIT)
```
Comment (mposolda) [inline quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakProcessor.java:105]: Is this change intentional? Wouldn't it have impact on the startup time?

Comment (pedroigor) [reply quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakProcessor.java:105]: Yes, it is ..

After analyzing the flame graph   for a startup I noticed stacks related to loading factory classes during static init. Where changing to runtime (main method) it does not show up. O noticed a slight improvement, thus the change.

Also, I think that initialization makes more sense in the main method after all configuration is processed during static init.

Comment (mposolda) [reply quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakProcessor.java:105]: Ok, Thanks!

**Review by mposolda — APPROVED** [review]
